src/scss/includes/components/itemList.scss

```diff
@@ -1,42 +1,24 @@
 .itemList-item {
   background: $background;
-
-  &:not(:last-child):after {
-    display: block;
-    content: '';
-    height: 1px;
-    margin: 0 30px;
-    background-color: $separator_color;
-  }
 }
 
 .itemList--hover .itemList-item {
   position: relative;
 
   &:hover {
     background: $surface;
-
-    &:before {
-      content: '';
-      position: absolute;
-      width: 4px;
-      height: 100%;
-      background-color: #FF3B4A;
-    }
   }
 }
 
 @media (max-width: 640px) {
+
   .itemList {
     padding-bottom: 12px;
-    margin: 0 15px;
+    margin: 0;
```

Comment:
You should get the same result by just removing the `margin: 0` lines, as it's the default.